components/admin_console/admin_definition.jsx

```diff
@@ -2972,6 +2972,384 @@ export default {
                     ],
                 },
             },
+            experimental: {
+                schema: {
+                    id: 'ExperimentalSettings',
+                    name: t('admin.advance.experimental'),
+                    name_default: 'Experimental',
```

Comment:
I don't know if the admin schema functionality supports some sort of big "these are experimental" disclaimer, but that could be nice to have here. 0/5 though.

Reply:
I think the fact that these settings are under an "experimental" category should suffice.